### PR TITLE
It worked before, now styles need to be fixed after PR #48

### LINKED ISSUE #50: Linked event don't open in a new page on item click
Linked events should be opened in a new page once one of them is clicked.

### PR DESCRIPTION
**Pull request by Daniilmass**
resolves #50

### CODE REVIEW

File: src/styles/single-event-page.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+   /**
+    * Hero-image as background 
+    */
+  heroImageDiv: {
+    backgroundImage: `url(${hero})`,
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center center",
+    backgroundSize: "cover",
+    color: "#fff",
+    height:"20vw",
```
Comment (Kaemppi) [inline src/styles/single-event-page.ts:40]: Just curious: why vw unit? It is a relative unit to viewport width and can cause some strange behaviour specially when used as heights. Unit vh is preferred in heights which is the same, but relative to viewport height.

File: linkedevents/events.php [hunk 2]
```diff
@@ -1,13 +1,16 @@
 <?php
   foreach ($data->events as $event) {
     $result = "";
+    $siteUrlBase = "http://localhost/";
```
Comment (terolamminpaa) [inline linkedevents/events.php:4]: There is a way to get this information from wordpress database using a function so it doesn't need to be hardcoded.
Maybe one of these?
[site url](https://developer.wordpress.org/reference/functions/site_url/)
[home url](https://developer.wordpress.org/reference/functions/get_home_url/)

File: src/components/pages/SingleEventPage.tsx [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+/**
+ * Interface for breadcrumb items
+ */
+interface Breadcrumb {
+    label?: string;
+    link?: string;
+  }
+
+/**
+ * A component for basic layout footer contents
```
Comment (terolamminpaa) [inline src/components/pages/SingleEventPage.tsx:47]: Update tsdocs

File: src/components/pages/SingleEventPage.tsx [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+    const { fetchedContent } = this.state;
+
+    return (
+      <BasicLayout lang={ lang }>
+        <div className={ classes.heroImageDiv }>
+          <p className={ classes.heroText }>{ fetchedContent ? fetchedContent.name.fi || "Event" : "Event" }</p>
+        </div>
+        <div className={ classes.breadcrumb }>
+            <Breadcrumbs separator=">">
+                <Link color="inherit" href="/" onClick={() => {}}>
```
Comment (terolamminpaa) [inline src/components/pages/SingleEventPage.tsx:101]: if the onClick handler isn't mandatory here then we could remove it

**Review by terolamminpaa — CHANGES_REQUESTED** [review]

**Comment by Daniilmass** [comment]
PR to be closed as it has been moved to #67 one to avoid conflicts

**Comment by Daniilmass** [comment]
PR to be closed as it has been moved to #67 one to avoid conflicts

**Pull request closed without merging**